package.json

```diff
@@ -30,10 +30,11 @@
   "devDependencies": {
     "@actions/artifact": "^0.3.1",
     "@actions/core": "^1.2.4",
+    "@types/node": "^14.0.22",
```

Comment:
We shouldn't be using node types for Node 14, all actions run on Node 12.

Reply:
Good catch. Even though I have Node `12.16.1` on my machine. VSCode still pulled in the latest types 😒